tests/test_helpers.py

```diff
@@ -124,12 +144,17 @@ def test_mention_html(self):
 
         assert expected == helpers.mention_html(1, "the name")
 
-    def test_mention_markdown(self):
-        expected = "[the name](tg://user?id=1)"
-
-        assert expected == helpers.mention_markdown(1, "the name")
+    @pytest.mark.parametrize(
+        "test_str, expected",
+        [
+            ("the name", "[the name](tg://user?id=1)"),
+            ("under_score", "[under_score](tg://user?id=1)"),
```

Comment:
can you include the other chars here that MDV1 wants us to escape outside of entities?